### PR TITLE
[REVIEW] Add notebook testing

### PR DESCRIPTION
**Pull request by efajardo-nv**
Added notebook testing to gpu ci build.

### CODE REVIEW

File: ci/gpu/build.sh [hunk 1]
```diff
@@ -113,4 +113,6 @@ else
     echo -e "\nTOP 20 SLOWEST TESTS:\n"
     # Wrap in echo to prevent non-zero exit since this command is non-essential
     echo "$(${WORKSPACE}/ci/getGTestTimes.sh testoutput.txt | head -20)"
+
+    ${WORKSPACE}/ci/gpu/test-notebooks.sh
```
Comment (dillon-cullinan) [inline ci/gpu/build.sh]:
```suggestion
    ${WORKSPACE}/ci/gpu/test-notebooks.sh 2>&1 | tee nbtest.log
    EXITCODE=$? 
    python /${WORKSPACE}/ci/utils/nbtestlog2junitxml.py nbtest.log
```

As discussed in Slack to add the results JUnit to Jenkins, that way its easier for devs to parse out the failures in the build.

**Comment by dillon-cullinan** [comment]
rerun tests

**Comment by efajardo-nv** [comment]
Notebook testing fails on SSSP notebook.

**Comment by dillon-cullinan** [comment]
rerun tests

**Comment by dillon-cullinan** [comment]
rerun tests

**Pull request merged**